src/unittest/test_logging.cpp

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Depending on your preference, the SPDX license notice could be used here to replace this boilerplate.

Reply:
will do